graylog2-web-interface/src/routing/Routes.ts

```diff
@@ -60,6 +60,9 @@ const Routes = {
       show: (notificationId: string) => `/alerts/notifications/${notificationId}`,
     },
   },
+  PROMO: {
+    SECURITY: '/promo/security',
+  },
```

Comment:
I think we can leave '/promo' out, I see a small benefit in using the same route for the promo and the non-promo page. You can just link to `/security` and do not need to worry about the security license status.
It would also be similar to the "enterprise" routing.

Reply:
Fixed

Reply:
It looks like the route is still `/promo/security`.

Reply:
Resolved.